tests/lib/reset.sh

```diff
@@ -44,7 +59,11 @@ reset_classic() {
         systemctl start snapd.socket
 
         # wait for snapd listening
-        while ! printf "GET / HTTP/1.0\r\n\r\n" | nc -U -q 1 /run/snapd.socket; do sleep 0.5; done
+        EXTRA_NC_ARGS="-q 1"
+        if [[ "$SPREAD_SYSTEM" = fedora-* ]]; then
+            EXTRA_NC_ARGS=""
```

Comment:
The logic is the other way round. It drops `-q 1` only on Fedora as it ships with a different netcat implementation (not the -openbsd variant as Debian/Ubuntu do) which does not support `-q`.